packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/daemonset.yaml

```diff
@@ -26,9 +26,12 @@ spec:
           volumeMounts:
             - mountPath: /fluent-bit/etc/
               name: config
-            - mountPath: /run/log/journal
+            - mountPath: {{ .Values.systemdLogPath | default "/var/log/journal" }}
```

Comment:
> Centos requires the rancher selinux package to work

Does that mean even without SELinux enabled ? If so we have to figure out why.

Reply:
No, the rancher-selinux package only does anything if SELinux is enabled. https://github.com/rancher/rancher/issues/31309#issuecomment-819749224 is the background

Reply:
If you set selinux to permissive on centos it works fine but that was not the default state for me with centos8 on do.

Reply:
I think this line is fine, but we should document something along the lines of...

"If SELinux is set to `Enforcing`, which most Fedora and RHEL derivatives distributions do by default, then you must install the `rancher-selinux` RPM as well"

...although we probably want something cleaner than that :) 

I like the idea of having the default option be the Ubuntu path. It's safer than having an empty option IMO

Reply:
I'm a bit lost, did we test with selinux on and our new rpm installed ? https://github.com/rancher/rancher/issues/31309#issuecomment-819751532

Reply:
Yes this was tested on centos and ubuntu and was able to collect systemd logs on both.